packages/core/src/geometry/GeometrySystem.ts

```diff
@@ -78,6 +80,11 @@ export class GeometrySystem extends System
          * @readonly
          */
         this.managedBuffers = {};
+
+        /**
+         * shortcut for the buffer system..
+         */
+        this._buffer = this.renderer.buffer;
```

Comment:
Very bad name for variable, we have `buffer`'s everywhere here. Use bufferSystem, bufSystem or something like that.

Reply:
I think consistency is important here - all systems so far are short hand like this.  Out of scope for this one, but we could do a separate PR that converts all systems to be post fixed with System?

Reply:
I'm not going to block this on this account, but building on the consistency argument - caching a system is nowhere used. I don't think we should be caching dependency systems like this. I find it inelegant, but that may be me :)